rosdep/python.yaml

```diff
@@ -6965,6 +6965,13 @@ python3-retrying:
     pip:
       packages: [retrying]
   ubuntu: [python3-retrying]
+python3-rocker:
+  alpine: [rocker]
+  debian: [python3-rocker]
+  fedora: [python3-rocker]
```

Comment:
I don't think that we have RPMs currently for fedora or rhel yet either.

https://src.fedoraproject.org/